components/Org/OrgTabs.js

```diff
@@ -1,73 +1,58 @@
 import { Tabs } from 'antd'
 import PropTypes from 'prop-types'
 import React from 'react'
-import { FormattedMessage } from 'react-intl'
 import MemberSection from '../Member/MemberSection'
 import { ProfilePanel } from '../VTheme/Profile'
 import { OrgAboutPanel } from './OrgAboutPanel'
 import Html from '../VTheme/Html'
 import VTabs from '../VTheme/VTabs'
-const { TabPane } = Tabs
-
-const orgTab =
-  <FormattedMessage
-    id='orgTabs.about'
-    defaultMessage='About'
-    description='Tab label on OrgDetailsPage'
-  />
-
-const orgMemberTab =
-  <FormattedMessage
-    id='orgMembers'
-    defaultMessage='Members'
-    description='show opportunities list on volunteer home page'
-  />
-
-const orgInstructionTab =
-  <FormattedMessage
-    id='orgInstructions'
-    defaultMessage='Getting Started'
-    description='show opportunities list on volunteer home page'
-  />
+import { OrgHistoryPanel } from './OrgHistoryPanel'
+import { orgTab, orgMemberTab, orgInstructionTab, orgOffersTab, orgEditTab, orgHistoryTab } from './OrgTabs.messages'
 
-const orgOffersTab =
-  <FormattedMessage
-    id='orgOffers'
-    defaultMessage='Offers'
-    description='show opportunities list on volunteer home page'
-  />
-
-const orgEditTab =
-  <FormattedMessage
-    id='orgTabs.edit'
-    defaultMessage='Edit'
-    description='Tab label for org Editor panel on organisation page'
-  />
+const { TabPane } = Tabs
 
 // Warning do not try to group tabs under an isFlag TabPanes must be direct Children of Tabs.
-export const OrgTabs = ({ org, onChange, canManage, defaultTab, isAuthenticated }) => (
+export const OrgTabs = ({
+  org,
+  archivedOpportunities,
```

Comment:
this is being passed down two levels of component when it is already in Redux. 
its not necessary to do this. 
using either withArchivedOpportunities or useSelect in the panel itself you can get access to the data.

Reply:
Also resolved in my latest changes by moving redux stuff into the `<OrgHistoryPanel />`